core/common/src/main/java/alluxio/Constants.java

```diff
@@ -387,6 +387,7 @@
   public static final String SWIFT_USER_KEY = "fs.swift.user";
   public static final String SWIFT_TENANT_KEY = "fs.swift.tenant";
   public static final String SWIFT_API_KEY = "fs.swift.apikey";
+  public static final String SWIFT_PASSWORD_KEY = "fs.swift.password";
```

Comment:
@gpang I don't want apiKey anymore. Documentation page updated to use "password" key.
However those who use "apiKey" will not be affected, since it's still supported by the code.